ui/custom.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module "*.svg" {
+    const content: any;
+    export default content;
+  }
+declare module "*.png" {
```

Comment:
Prettier didn't run on these files?

Reply:
We'll need to eslint-disable the any here too, just search the codebase for `// eslint-disable-next-line` to see how it's done

Reply:
Oh ok there seems to be two declaration files, this one `ui/custom.d.ts`, and then `ui/src/global.d.ts`.

Let's get rid of one and choose the other (im in favour of `global.d.ts`)

Reply:
Going to keep the `ui/custom.d.ts` (+rename it) and delete the other custom types file. `global.d.ts` in /src doesn't seem to be resolving the svg module errors.